### DISCUSSION TITLE
Help calculating angular distributions in U235 / U238 with ENDFVIII data

### DISCUSSION THREAD

**Discussion by sombrereau** [njoy/NJOY21#141] [Q&A]
NJOY team (@whaeck / @nathangibson14 ),

I'm a ~~very~~ new NJOY user. I'm trying to generate the covariance data with NJOY to calculate the nuclear data uncertainty for a potential Uranium Fueled Advanced Nuclear Reactor.

I'm getting errors when I try to process the ENDFVIII U235 / U238 nuclear data to create tapes for covariance of angular distribution.

NJOY issues the following message:
> ***error in covcal***lb= 2 when lt= 1

I've tried to include every reaction in the endf data in my input and also tried to run test 15 (with the [ENDFVIII nuclear data](https://www-nds.iaea.org/public/download-endf/ENDF-B-VIII.0/n/n_9228_92-U-235.zip)), and I've tried to use the [preliminary INDEN ENDF evaluation with updated covariance data](https://www-nds.iaea.org/INDEN/data/u235ib46o28t6DNcnu5ef0STzn_ENDF.zip)

I'm running NJOY21 v. 1.1.1. I'm using the [attached input file](https://github.com/njoy/NJOY21/files/5756089/endf8_covariance_U235_covariance_mf34.txt).

Thanks for any tips,

-Tommy Cisneros

**Comment by whaeck** [comment]
This appears to be a violation of the ENDF format. Section 33.2.2.2 of the ENDF manual states:
```
For LB=0, 1 and 2 we have LT=0, i.e., only one Ek table. For LB=3 and LB=4 we have LT != 0, i.e., two E-tables, the Ek and the El-tables.
```
ERRORR appears to be strictly enforcing this rule. Whether this is an evaluation error, or ERRORR being unnecessarily pedantic, I cannot say. Either way, you should talk to the evaluator to get a corrected evaluation for your INDEN evaluation.

I did not run the input so I would like to ask if this happens for ENDF/B-VIII as well?

**Reply by sombrereau** [reply]
Thanks for looking at this Wim,

I just double checked with the ENDFVIII data and the INDEN eval again -- I'm still getting those errors : \
